apps/web/app/(app)/[emailAccountId]/assistant/RuleForm.tsx

```diff
@@ -1226,6 +1240,68 @@ function ActionCard({
           {/* Show the variable pro tip when action has visible fields that can use variables */}
```

Comment:
we can drop this comment. feel like ai likes to comment the code, but we don't need it a lot of the time